### DISCUSSION TITLE
File upload size limit

### DISCUSSION THREAD

**Discussion by shivshankardayal** [ntex-rs/ntex#415] [Q&A] [answered]
I have seen the file upload example in examples repo. How can I limit its size?

**Comment by shivshankardayal** [comment]
@fafhrd91 Can you please help?

**Reply by fafhrd91** [reply, marked as answer]
there are several options:

* you can get request payload manually like [this](https://github.com/ntex-rs/examples/blob/a3cb7646348f0ba25efffc639125eec766860368/json/src/main.rs#L30)
* you can limit of "Bytes", like in [this example](https://github.com/ntex-rs/examples/blob/a3cb7646348f0ba25efffc639125eec766860368/json/src/main.rs#L48) and [configuration](https://github.com/ntex-rs/examples/blob/a3cb7646348f0ba25efffc639125eec766860368/json/src/main.rs#L69), just use [PayloadConfig](https://docs.rs/ntex/2.5.0/ntex/web/types/struct.PayloadConfig.html)

**Reply by shivshankardayal** [reply]
Much thanks. I will check these out.
